idf_build_apps/manifest/manifest.py

```diff
@@ -45,7 +45,7 @@ def __init__(self, stmt: str, temporary: bool = False, reason: t.Optional[str] =
         try:
             self.stmt: BoolStmt = parse_bool_expr(stmt)
             self._stmt: str = stmt
-        except (ParseException, InvalidIfClause) as ex:
+        except (ParseException, InvalidIfClause, AttributeError) as ex:
```

Comment:
let's catch all exceptions instead?


```suggestion
        except Exception as ex:
```